planning/behavior_velocity_planner/autoware_behavior_velocity_blind_spot_module/src/util.cpp

```diff
@@ -389,10 +388,10 @@ std::optional<lanelet::CompoundPolygon3d> generate_attention_area(
       ? blind_side_lanelets_before_turning_merged.leftBound()
       : blind_side_lanelets_before_turning_merged.rightBound();
   for (const auto & point : blind_side_lanelet_boundary_before_turning) {
-    far_side_boundary.push_back(remove_const(point));
+    far_side_boundary.push_back(autoware::experimental::lanelet2_utils::remove_const(point));
```

Comment:
As `using autoware::experimental::lanelet2_utils::remove_const;` is declared in this function, I guess
```suggestion
    far_side_boundary.push_back(remove_const(point));
```

Reply:
And also other parts are too in this function `generate_attention_area()`